### PR TITLE
Code review

### LINKED ISSUE #1: Code and documentation review

### PR DESCRIPTION
**Pull request by DanielaIvanova**
I tested the code and overall looks good to me.
I made some code refactoring. Also formatted the code.
Examples should be added in Protocol section, in order to be clear how the users can communicate with the server.
Would be nice if we have some tests, currently the auto-generated tests are not working.
The output in the page is not placed correctly.
It would be great if we had a CI tool in the repo.

Regarding documentation, I made some changes and I removed the `NOTE` for old rocksdb and I opened an issue in [AEREPL](https://github.com/aeternity/aerepl/issues/13) project, because this issue has nothing to do with this project.

**NOTE:**
For some reason when we get the dependancies and compile them, the `AEREPL` artefacts are not present in root `_build/` folder. Updating the environment with `export ERL_LIBS=$ERL_LIBS:$(pwd)/deps/aerepl/_build/default/lib` will work only after first failed compilation. It's does't look ok and consistent. In my opinion fixes should be done to `AEREPL` repo.

To auto-format the code(you have to be in the root directory):
`mix format`

resolves #1

### CODE REVIEW

File: lib/aerepl_http/aerepl_utils.ex [hunk 1]
```diff
@@ -1,38 +1,31 @@
 defmodule ReplUtils do
+  def render(state, repl_str), do: List.to_string(:aere_repl.render_msg(state, repl_str))
```
Comment (tolbrino) [inline lib/aerepl_http/aerepl_utils.ex:2]: Better use https://elixirschool.com/en/lessons/basics/pipe-operator/ for such constructs.

File: mix.exs [hunk 2]
```diff
@@ -35,9 +35,9 @@ defmodule AereplHttp.MixProject do
     [
       {:phoenix, "~> 1.4.16"},
       {:phoenix_pubsub, "~> 1.1"},
-#      {:phoenix_ecto, "~> 4.0"},
-#      {:ecto_sql, "~> 3.1"},
-#      {:postgrex, ">= 0.0.0"},
+      # {:phoenix_ecto, "~> 4.0"},
```
Comment (tolbrino) [inline mix.exs:38]: Just remove the lines.

File: README.md [hunk 3]
```diff
@@ -25,14 +37,14 @@ The server awaits for incoming connections and assigns a session key to each use
...
-   * `ask` indicates that the REPL as asked a question. It should provide the information about valid answers and the default option. If the user replies with empty input the default option will be chosen. If the input won't match with any of the options the question will be re-asked.
+Where:
+ - `key` describes the session key /TODO: should be a cookie actually/
+ - `output` is the message to be printed out
+ - `status` tells the type of the response:
+   - `success` is a regular output with successful outcome
+   - `error` means the query wasn't processed at all because of some mistake on the user side. This includes for example type errors, bad syntax, misusing the REPL features etc.
+   - `internal error` is the error on the REPL side. The output will probably contain the REPL stacktrace which will be a valuable information if reported to the developers.
+   - `ask` indicates that the REPL was asked a question. It should provide the information about valid answers and the default option. If the user replies with empty input the default option will be chosen. If the input won't match with any of the options the question will be re-asked.
```
Comment (radrow) [inline README.md:47]:
```suggestion
   - `ask` indicates that the REPL has asked a question. It should provide the information about valid answers and the default option in the `output` field. If the user replies with empty input the default option will be chosen. If the input won't match with any of the options the question will be re-asked.
```
The question is being asked by the REPL

**Comment by AleksandarFilipov** [comment]
Friendly tip, reviewing formatter changes is no fun, so keeping those changes in a separate commit makes the reviewers life easier.

**Comment by AleksandarFilipov** [comment]
Btw, this project looks really promising :)

**Pull request merged**